tests-functional/clients/status_backend.py

```diff
@@ -109,33 +111,6 @@ def wait_for_healthy(self, timeout=10):
     def health(self):
         return self.api_request("health", data=[], url=self.base_url, quiet=True)
 
-    def api_request(self, method, data, url=None, quiet=False):
-        url = url if url else self.api_url
-        url = f"{url}/{method}"
-        if not quiet:
-            logging.debug(f"Sending POST request to url {url} with data: {json.dumps(data, sort_keys=True)}")
-        response = requests.post(url, json=data)
-        if not quiet:
-            logging.debug(f"Got response: {response.content}")
-        return response
-
-    def verify_is_valid_api_response(self, response):
-        assert response.status_code == 200, f"Got response {response.content}, status code {response.status_code}"
-        assert response.content
-        logging.debug(f"Got response: {response.content}")
-        try:
-            error = response.json()["error"]
-            assert not error, f"Error: {error}"
```

Comment:
If we agree to remove this as automatic check, then we should add `assert not response.error` to all tests, right?

Reply:
But as I said, I'm not sure if it's the right approach to pollute all tests with such asserts.
Hope you know what's the right way, maybe it is better to have some `check_error` argument, dunno.